addons/product/data/product_demo.xml

```diff
@@ -600,6 +626,21 @@
             <field name="image_1920" type="base64" file="product/static/img/product_product_d01-image.jpg"/>
         </record>
 
+        <record id="product_template_no_variant_attribute" model="product.template">
+            <field name="name">Outdoor dinning table</field>
+            <field name="categ_id" ref="product_category_outdoor_furniture"/>
+            <field name="list_price">589</field>
+            <field name="detailed_type">consu</field>
+            <field name="image_1920" type="base64" file="product/static/img/product_template_no_va-image.png"/>
+        </record>
+
+        <!-- the product template attribute lines have to be defined before creating the variants -->
```

Comment:
I don't think this comment is really useful, as it is already present in the file
```suggestion
```